contracts/token/L2ECO.sol

```diff
@@ -100,48 +159,83 @@ contract L2ECO is ERC20Upgradeable, EIP712Upgradeable, IERC165 {
         tokenRoleAdmin = _l2Bridge;
     }
 
-    /** Access function to determine the token balance held by some address.
+    /** 
+     * @dev Access function to determine the token balance held by some address.
```

Comment:
inflation corrected

Reply:
tokens = thing that are inflated

Reply:
so no, tokens is the right thing to say here